make-coe/make-coe.go

```diff
@@ -117,7 +138,7 @@ func (carina *MakeCOE) ListClusterTemplates() ([]common.ClusterTemplate, error)
 
 	results, err := carina.listClusterTypes()
 	if err != nil {
-		return nil, err
+		return nil, handleLibcarinaError(err)
```

Comment:
Oh, I did do that. I just forgot to delete this wrapped call, so it got handled twice. Whoops!